source/_integrations/zha.markdown

```diff
@@ -581,6 +581,29 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
   - Place your Zigbee Coordinator away from any Wi-Fi access points and all other sources of WiFi.
   - Wi-Fi frequency ranges can overlap with Zigbee, see the section above on defining Zigbee channel use.
 
+### Zigbee network visualization in ZHA UI
+
+ZHA configuration UI has a window/tab for *Visualization* that visually represents device links in your Zigbee network topology.
+
+The network visualization can help get a quick overview in identifying devices reporting links with poor connection (that is, low values). However, be aware that you will need to look at the ZHA logs instead for deeper troubleshooting.
+
+The visualization shows multi-hop connections between your paired devices and their reported cumulative values of Received Signal Strength Indicator (RSSI) and Link Quality Indication (LQI).
+
+Note that the exact method in which these values are reported depends on the Zigbee network stack used on each device, and LQI values can be modified at each step as the massage propagates through the mesh networking matrix.
+
+#### How to interpret RSSI and LQI values
+
+Interpreting RSSI and LQI values is complicated and the recommendation is that normal users should ignore them unless guided by Zigbee troubleshooting specialists. If you do delve into this then it is important to understand not to judge only RSSI or LQI values on their own separately as it is really the combination of these two that needs to interpret be when troubleshooting if Zigbee messages are being dropped.
+
+RSSI (Received Signal Strength Indicator) values are simple to interpret as they are just an indicator value of the raw signal strength between two devices. RSSI values are shown as negative numbers represented in -dBm format (0 to -100 power ratio in decibels of the measured power referenced to one milliwatt). Lower negative values are better as they indicate less interference and a good signal. It is important to understand that RSSI information is only between the endpoint device and the first hop from that device, as such it may not necessarily show signal strength to the Zigbee Coordinator but instead could be showing signal strength to the nearest Zigbee Router device.
```

Comment:
```suggestion
RSSI (Received Signal Strength Indicator) values are an indicator value of the raw signal strength between two devices. RSSI values are negative numbers in -dBm format (0 to -100 power ratio in decibels of the measured power referenced to one milliwatt). Lower negative values indicate less interference and a good signal. RSSI information is only between the endpoint device and the first hop from that device. As such it may not necessarily show signal strength to the Zigbee Coordinator but instead could be showing signal strength to the nearest Zigbee Router device.
```